### DISCUSSION TITLE
Welcome to kerykeion Discussions!

### DISCUSSION THREAD

**Discussion by g-battaglia** [g-battaglia/kerykeion#61] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 7flash** [comment]
Hi @g-battaglia thank you for amazing work, the most reliable library we could find among alternatives

At chatgod.space we are using kerykeion python library to calculate sky bodies coordinates given user birth time, then we derive human design pre/natal charts based on those coordinates using my library https://github.com/7flash/i-ching-astrology , and then we feed relevant to user data into AI communicating with user about his profile.
